anagramGame.py

```diff
@@ -19,6 +23,13 @@
 print("                                                            Avengers Jumble Game                                                    ")
 print("*"*180)
 print()
+if t<12:
+    print("Good morning",name)
+elif 12 <= t < 18:
+    print("Good Afternoon",name)
+else:
+    print("Good Evening",name)    
+    
```

Comment:
You are Actually printing your greetings whereas you can use i.say greetings also.